cluster/gce/upgrade.sh

```diff
@@ -102,6 +100,10 @@ function wait-for-master() {
 function prepare-upgrade() {
   ensure-temp-dir
   detect-project
+  # TODO(mbforbes): Do we still need get-password?
```

Comment:
Follow same argument as my lengthy comment above about the bearer token. (And I have the same question—if it's indeed not needed, we need to change some stuff.)